### ISSUE TITLE
Non-Vue projects shouldn't need to pull in Vue as a required dependency

### ISSUE THREAD

**Issue by ecbrodie** [gvaldambrini/storybook-router#22]
### Which version are you using?

0.3.2

### Are you using storybook-router with a react based project or a vue one?

React

### Please describe the problem:

I am working on a project that just uses React. As such, I have no use for having Vue as a dependency.

Unfortunately, my Yarn package manager will pull in `vue` as a dependency regardless, since it is a dependency in `storybook-router`'s `package.json` file. I now have `vue` in my project's `node_modules`, despite the fact that there's no way my `react` project will ever use that library. I wonder if it makes more sense to have `vue` specified as a _peer dependency_ in `package.json`. That way, it is up to the app that uses `storybook-router` to determine whether `vue` should actually be installed into `node_modules` as a dependency.

I feel like the same can be said in vice-versa: an app that only uses `vue` will have no reason to pull in `react` as a dependency.

**Comment by gvaldambrini** [comment]
Hi @ecbrodie, there is already a work in progress task which will split the npm package so that the react version doesn't have vue as dependency and viceversa. Unfortunately the solution of using the `peerDependencies` doesn't work very well (for more details on this, please read #19).

**Comment by ecbrodie** [comment]
I'm happy to see that there is already WIP to get this fixed, and also that I'm not going to experience one of those Big Lebowski "Am I the only one here that thinks this is a legitimate issue" moments.

**Comment by PierreGUI** [comment]
Hey @gvaldambrini awesome work! Here is something related to this issue that could maybe help: [storyshots: Why don't we install dependencies of each framework ?](https://github.com/storybooks/storybook/tree/master/addons/storyshots#why-dont-we-install-dependencies-of-each-framework-)

TLDR: there's no easy way to achieve this, let people install the required dependencies themselves.

**Comment by gvaldambrini** [comment]
The new version of the packages have been published, please upgrade your `package.json`:
https://www.npmjs.com/package/storybook-vue-router

**Comment by ndelangen** [comment]
Awesome @gvaldambrini !
